packages/webchannel-wrapper/src/index.d.ts

```diff
@@ -44,13 +97,23 @@ export interface WebChannelOptions {
   fastHandshake?: boolean;
   disableRedac?: boolean;
   clientProfile?: string;
-  internalChannelParams?: { [key: string]: boolean | number };
+  internalChannelParams?: {
+    forwardChannelRequestTimeoutMs?: number;
+  };
   xmlHttpFactory?: unknown;
   requestRefreshThresholds?: { [key: string]: number };
 }
 
+export interface WebChannel {
+  open(): void;
+  close(): void;
+  halfClose(): void;
+  listen<T>(type: EventType, cb: (param: T) => void): void;
+  send<T>(msg: T): void;
```

Comment:
nit: Is there a benefit to making these generic instead of typing them as `unknown` or similar?

Reply:
`Send` can be unknown (it should probably not be - but then we should both the generic type on WebChannel). I made it unknown for now.
The <T> in `listen` calls back to us so it cannot be unknown.